deps/Makefile

```diff
@@ -658,6 +658,12 @@ $$(LLVM_SRC_DIR)/$1.patch-applied: $(LLVM_SRC_DIR)/configure | $$(SRCDIR)/$1.pat
 	echo 1 > $$@
 LLVM_PATCH_LIST += $$(LLVM_SRC_DIR)/$1.patch-applied
 endef
+define LLVM_PATCH0
+$$(LLVM_SRC_DIR)/$1.patch-applied: $(LLVM_SRC_DIR)/configure | $$(SRCDIR)/$1.patch
+	cd $$(LLVM_SRC_DIR) && patch -p0 < $$(SRCDIR)/$1.patch
```

Comment:
Modifying the existing patches would lead to the `.patch-applied` files indicating an incorrect state in everyone's existing build, so the new modifications would not get applied except in from-scratch builds of llvm.